opta/amplitude.py

```diff
@@ -73,7 +73,9 @@ def send_event(
             return
         event_properties = event_properties or {}
         user_properties = user_properties or {}
-        insert_id = "".join(random.choices(string.ascii_letters + string.digits, k=16))
+        insert_id = "".join(
+            random.choices(string.ascii_letters + string.digits, k=16)  # nosec
+        )  # nosec
```

Comment:
```suggestion
        insert_id = "".join(
            random.SystemRandom().choices(string.ascii_letters + string.digits, k=16)
        )
```

See my comment here: https://github.com/run-x/opta/pull/497#discussion_r752931269. Looks like a change to the security tool we use in CI changed which is why its now complaining about this line.